addons/knowledge/static/src/webclient/commands/knowledge_providers.js

```diff
@@ -0,0 +1,162 @@
+/** @odoo-module */
+
+
+import { _lt } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+import { HotkeyCommandItem } from "@web/core/commands/default_providers";
+import { splitCommandName } from "@web/core/commands/command_palette";
+
+const { Component, xml } = owl;
+
+// Articles command
+class KnowledgeCommand extends Component {}
+KnowledgeCommand.template = xml`
+    <div class="o_command_left d-flex align-items-center">
```

Comment:
this template, and the other in this file should not be inlined in js. don't forget that if you do that, they will not be translated (nor modifiable by xpaths). also, I think that you could move the components in their own file, next to this one